Main.java

```diff
@@ -3,74 +3,121 @@
             return false;
         }
         return true;
     }
 
     private static void wordCount(String textFilePath, String searchKeyword) {
 
-        StringBuilder stringBuilder = new StringBuilder();
-        int wordCount = 0;
+        int count = 0;
+        String errorMessage;
         String fileData;
         try {
             BufferedReader bufferedReader = Files.newBufferedReader(Paths.get(textFilePath));
-            fileData = bufferedReader.readLine();
-            if (fileData == null) {
-                System.out.println("File does not contain any data");
-                return;
-            } else {
-                while (fileData != null) {
-                    stringBuilder.append(fileData).append("\n");
-                    fileData = fileData.replaceAll(REMOVE_SPECIAL_CHAR, SINGLE_SPACE);
-                    StringTokenizer splitIntoToken = new StringTokenizer(fileData);
-                    while (splitIntoToken.hasMoreTokens()) {
-                        if (searchKeyword.equalsIgnoreCase(splitIntoToken.nextToken())) {
-                            wordCount++;
-                        }
+            while ((fileData = bufferedReader.readLine()) != null) {
+                fileData = fileData.replaceAll(REMOVE_SPECIAL_CHAR, SINGLE_SPACE);
+                StringTokenizer splitIntoToken = new StringTokenizer(fileData);
+                while (splitIntoToken.hasMoreTokens()) {
+                    if (searchKeyword.equalsIgnoreCase(splitIntoToken.nextToken())) {
+                        count++;
                     }
                 }
-                if (wordCount == 0) {
-                    System.out.println("Word does not exist");
-                } else {
-                    System.out.println("Word found");
-                    System.out.println(searchKeyword + " occurs " + wordCount + " times in the file");
-                }
+            }
+            if (count == 0) {
+                errorMessage = "Word does not exist";
+                System.out.println(errorMessage);
+                Database.databaseOperation(textFilePath, searchKeyword, searchResult = "Failure", errorMessage, count);
+            } else {
+                System.out.println("Word found");
+                System.out.println(searchKeyword + " occurs " + count + " times in the file");
+                Database.databaseOperation(textFilePath, searchKeyword, searchResult = "Success", null, count);
             }
         } catch (Exception e) {
             e.printStackTrace();
         }
     }
 }
 
+class Database {
+
+    private static String CONNECTION_URL = "jdbc:mysql://localhost:3306/elixr_labs_internship";
+    private static String USER = "root";
+    private static String PASSWORD = "abhijith.sajimon@29";
+
+    public static void databaseOperation(String textFilePath, String searchKeyword, String searchResult, String errorMessage, int count) {

```

Comment:
Method name databaseOperations is too generic. Please use a method name that explains what does the method do in short.

Reply:
Sure. Will do.